ocaml/idl/datamodel_repository.ml

```diff
@@ -223,8 +247,15 @@ let t =
           "The file name of the GPG public key of this repository"
       ; field ~qualifier:StaticRO ~lifecycle:[] ~ty:origin "origin"
           ~default_value:(Some (VEnum "remote"))
-          "The origin of the repository. 'remote' if the origin of the \
+          "The origin of this repository. 'remote' if the origin of the \
            repository is a remote one, 'bundle' if the origin of the \
-           repository is a local bundle file."
+           repository is a local bundle file, 'remote_pool' if the origin of \
+           the repository is a remote pool"
+      ; field ~qualifier:StaticRO ~lifecycle:[] ~ty:String
+          ~default_value:(Some (VString "")) "address"
+          "The hostname of the host which hosts this repository"
+      ; field ~qualifier:StaticRO ~lifecycle:[] ~ty:String
```

Comment:
I'm not sure if the `certificate` should be `RW` as the certificate may be changed for refreshing.
But anyway a user can remove, re-introduce, re-sync the repository. This might not be a problem.